src/Integration/ModulesIntegration.php

```diff
@@ -63,11 +65,33 @@ public static function applyToEvent(self $self, Event $event): void
     private function processEvent(Event $event): void
     {
         if (empty(self::$loadedModules)) {
-            foreach (Versions::VERSIONS as $package => $rawVersion) {
-                self::$loadedModules[$package] = PrettyVersions::getVersion($package)->getPrettyVersion();
+            foreach (self::getInstalledPackages() as $package) {
+                try {
+                    self::$loadedModules[$package] = PrettyVersions::getVersion($package)->getPrettyVersion();
+                } catch (\OutOfBoundsException | VersionMissingExceptionInterface $exception) {
```

Comment:
Would it makes sense to catch any `Throwable`? Since we're inside an integration, if something fails it's really difficult to know what went wrong. In addition to that, regardless of what happens, if we don't report the version of a certain package it's not the end of the world

Reply:
I agree. I also ported the same change into the original PR #1170 too with 43bee01.